### DISCUSSION TITLE
Pcap files are not showing during a drill down and selecting the PCAP

### DISCUSSION THREAD

**Discussion by gbrodt** [Security-Onion-Solutions/securityonion#4187] [Unsupported Versions]
This is happening with a student I'm working with and other identical student pods do not have the issue  2.3.40
The pcap files are on the disk but the pcap job returns no results (for everything). It's like the connection to the pcaps has been lost. 
![Caps-001303](https://user-images.githubusercontent.com/75701987/118142289-d9496480-b3cf-11eb-9801-14682dee34c4.jpg)
![Caps-001300](https://user-images.githubusercontent.com/75701987/118142301-dc445500-b3cf-11eb-802a-34be6a6f7cf0.jpg)

**Comment by jertel** [comment]
Check the /opt/so/log/sensoroni/sensoroni.log on the NLSO minion. Look for errors or some indication that it is aware of a job 1014.

**Reply by gbrodt** [reply]
This is about all I see, not to helpful. 

![Caps-001306](https://user-images.githubusercontent.com/75701987/118291415-33642b80-b49d-11eb-8475-7ec6d3d89364.jpg)

**Reply by jertel** [reply]
Ok take a look at /opt/so/log/soc/sensoroni-server.log on the manager node and let's see if we can find out why it's returning a 500 error to that minion request.

**Reply by Mastadamus** [reply]
I am having what could be the exact problem to a tee.  PCAPS just stay pending no matter if job was generated from within SOC or kibana.  I have a standalone build running on the CentOs Iso. Note this didn't happen till after the 2.3.50 update via soup.  Let me know if you'd like me to make a new post. 
![FailedtoPoll](https://user-images.githubusercontent.com/79027361/118419323-4172a180-b681-11eb-8791-236ddd3b1348.JPG)

**Reply by gbrodt** [reply]
This is also a stand alone: Here is what I'm seeing in that log for errors (had to dig a bit - thanks grep). 

![image](https://user-images.githubusercontent.com/75701987/118495886-3eab9700-b6e9-11eb-8783-f7cb7e2f165a.png)

**Reply by jertel** [reply]
The "Pivot to PCAP" link in Kibana does not work for all events. Since the error in your log file claims that no Zeek record was found for that UID it leads me to believe that you're running into that limitation. Though I would not have expected the PCAP job to get created in that scenario. So it's also possible that the log portion you pasted is not correlated to the PCAP Job in "Pending" status. If you click on the Job it should give you more information about it which can help you correlate back to the log files.

**Reply by gbrodt** [reply]
The issues is that it's everything! The are no pcaps accessible under any events, but there are files on the disk.

**Reply by jertel** [reply]
Tracking in #4220.
